server/data/educationAndWorkPlanApiMapper.ts

```diff
@@ -0,0 +1,22 @@
+import type { CreateGoalRequest, CreateStepRequest } from 'educationAndWorkPlanApiClient'
+import type { CreateGoalDto } from 'dto'
+
+export default class EducationAndWorkPlanApiMapper {
+  createGoalDtoToCreateGoalRequest(createGoalDto: CreateGoalDto): CreateGoalRequest {
```

Comment:
Rather than making this a class, I wonder if you'd be better off doing away with the class, and simply exporting the function, and naming the source file the same
Which makes me wonder what the function name should be? 🤔 

How about:
`educationAndWorkPlanApiMappers.ts` (plural, because perhaps it exports several functions that can map stuff relating to this API ?)
And then its contents might be something like:
```
const mapCreateGoalDtoToCreateGoalRequest(createGoalDto: CreateGoalDto): CreateGoalRequest {
...
...
}

export default {
  mapCreateGoalDtoToCreateGoalRequest,
}
```
(and then you could have several functions in the module (source file), all being exported as a named function)

Which then makes the Client class below a little easier in respect of using it (and mocking it in its tests)

Reply:
Nice idea. I did wonder about introducing different mapper files and where to put them :)